actions/transactions/broadcast_callback.go

```diff
@@ -8,15 +8,14 @@ import (
 )
 
 // broadcastCallback will handle a broadcastCallback call from the broadcast api
-// Broadcast Callback godoc
-// @Summary		Endpoint designed for receiving callbacks from Arc (service responsible for submitting transactions to the BSV network)
-// @Tags		Transactions
-// @Param 		transaction body broadcast.SubmittedTx true "Transaction"
-// @Success		200
-// @Failure		400	"Bad request - Error while parsing transaction from request body"
-// @Failure 	500	"Internal Server Error - Error while updating transaction"
-// @Router		/transaction/broadcast/callback [post]
-// @Security	callback-auth
+// Summary		Endpoint designed for receiving callbacks from Arc (service responsible for submitting transactions to the BSV network)
+// Tags		Transactions
+// Param 		transaction body broadcast.SubmittedTx true "Transaction"
+// Success		200
+// Failure		400	"Bad request - Error while parsing transaction from request body"
+// Failure 	500	"Internal Server Error - Error while updating transaction"
+// Router		/transaction/broadcast/callback [post]
+// Security	callback-auth
```

Comment:
IMHO you can/should remove those comments entirely (eventually add a note, that we don't want document it in swagger.

Reply:
Fixed. I removed the swaggo comments.